packages/testsuite/cypress/support/containers-utils.ts

```diff
@@ -25,6 +45,36 @@ declare global {
        * @category Containers util
        */
       startWildflyContainer(): Chainable<unknown>;
+      /**
+       * Start a Wildfly container. This typical command which needs to executed in before method in every TC.
+       *
+       * This container will have a secured management API.
```

Comment:
I suggest to highlight that standard standalone.xml is used, and host network mode is used (like in case of Keycloak).

Reply:
now it looks like:

       * Start a Wildfly container wich is secured. This is not for a common use.
       *
       * This container is using standart "standalone.xml" and have a secured management interface. The container is 
       * running in network mode "host". To enable communication between browser (not in container), 
       * application server (in container), and Keycloak (in container). Those need to use the same network 
       * and consistent host names. This mean the wildlfly looks like it is stared directly on a host machine.
       * There can be potention conflict for used ports so there is range from 8080 to 8180 to check the port
       * free and then is calculated port offset for start parameter (0-100).